Pod/Classes/MPOFaceServiceClient.m

```diff
@@ -201,6 +235,15 @@ - (NSURLSessionDataTask *)findSimilarWithFaceId:(NSString *)faceId faceIds:(NSAr
     }];
 }
 
+- (NSURLSessionDataTask *)findSimilarWithFaceId:(NSString *)faceId faceIds:(NSArray *)faceIds maxNumOfCandidatesReturned:(NSInteger)maxNumOfCandidatesReturned mode:(MPOSimilarFaceSearchingMode)mode completionBlock:(MPOSimilarFaceArrayCompletionBlock)completion {
+    return [self findSimilarWithFaceId:faceId faceListId:nil faceIds:faceIds maxNumOfCandidatesReturned:20 mode:mode completionBlock:completion];
```

Comment:
20 is the default value. if user specify maxNumOfCandidatesReturned : 10. Will 10 be transferred to the findSimilarWithFaceId function?